File: PolicyManagement.Api/PolicyManagement.Models/Motor/PaymentFormDataModel.cs
using PolicyManagement.Dtos.Common;
using System;

namespace PolicyManagement.Models.Motor
{
    public class PaymentFormDataModel
    {
        public short Mode { get; set; }
        public int Amount { get; set; }
        public string InstrumentNumber { get; set; }
        public string DatedString { get; set; }
        public DateTime? Dated { get; set; }
        public DateDto DatedDto { get; set; }
        public short Bank { get; set; }
    }
}

File: PolicyManagementAngular/src/app/app-entites/models/motor/payment-form-data-model.ts
import { IDateDto } from "src/app/app-entites/dtos/common/date-dto";

export interface IPaymentFormDataModel {
    Mode: number;
    Amount: number;
    InstrumentNumber: string;
    DatedString: string;
    DatedDto: IDateDto|null;
    Bank: number;
}